jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -350,6 +350,15 @@ public JibContainerBuilder createJibContainerBuilder(
     return new DirectoryWalker(Paths.get(project.getBuild().getOutputDirectory())).walk().asList();
   }
 
+  @Override
+  public List<Path> getDependencies() {
+    return project
+        .getArtifacts()
+        .stream()
+        .map(artifact -> artifact.getFile().toPath())
```

Comment:
`GradleProjectProperties` also checks if the dependency is a jar whereas this does not. Is this intentional?

Reply:
Good point. I am not sure if all the artifacts from `project.getArtifacts()` will always be `.jar` files, but I think usually they will be `.jar` files for typical Java projects.

I've checked the `MavenProjectProperties` code gain, and I can tell at least that we always put all the files from `project.getArtifacts()` into `/app/libs`. (All of them are added as dependencies and only them.)

https://github.com/GoogleContainerTools/jib/blob/a720aa1fb9d86c3a7115544ee4e3d59602361e12/jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java#L291-L299

([`classifyDependencies(...)`](https://github.com/GoogleContainerTools/jib/blob/a720aa1fb9d86c3a7115544ee4e3d59602361e12/jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java#L334-L344) classifies `project.getArtifacts()` into project, snapshot, and third-party dependencies.)

So, doing this should match the actual dependencies we put.

Reply:
And for Gradle, checking `.jar` is a bit different to obtain dependencies than we current do in `GradleProjectProperties.createJibContainerBuilder()`; it doesn't strictly follow the logic there. However, I think this should work.